compiler/src/main/scala/dx/compiler/WorkflowCompiler.scala

```diff
@@ -167,6 +177,168 @@ case class WorkflowCompiler(extras: Option[Extras],
     (commonMeta ++ workflowMeta, commonDetails)
   }
 
+  private def listMd(s: String): String = {
+    "\n* " + s
+  }
+
+  private def listMd2(s: String): String = {
+    "\n    * `" + s + "`"
+  }
+
+  // Summarize reportable dependencies from details of a workflow
```

Comment:
Add a TODO that all this reporting code should be moved to templates.

Reply:
Added TODO comment